plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraClientConfig.java

```diff
@@ -104,7 +103,7 @@ public void testExplicitPropertyMappings()
                 .put("cassandra.no-host-available-retry-timeout", "3m")
                 .put("cassandra.speculative-execution.limit", "10")
                 .put("cassandra.speculative-execution.delay", "101s")
-                .put("cassandra.protocol-version", "V2")
+                .put("cassandra.protocol-version", "V3")
```

Comment:
I understand that V2 no longer works, am I right? If so we should not allow users to set this.

Reply:
Indeed @kokosing . Dropping the support for V2 of the protocol was a design decision from Datastax


Please see also:  https://github.com/trinodb/trino/issues/7729#issuecomment-826836017